packages/@react-stately/selection/src/useMultipleSelectionState.ts

```diff
@@ -27,7 +43,8 @@ export interface MultipleSelectionStateProps extends MultipleSelection {
 export function useMultipleSelectionState(props: MultipleSelectionStateProps): MultipleSelectionState {
   let {
     selectionMode = 'none' as SelectionMode,
-    disallowEmptySelection
+    disallowEmptySelection,
+    allowDuplicateSelectionEvents
```

Comment:
technically this has been the default behavior, are we sure we want to change the default?

Reply:
IMO the previous behavior felt like buggy behavior since typically you'd expect a selection change event if selection is actually changing.

Reply:
I'd argue that it was a bug

Reply:
great, i'm in agreement as well